docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Create a KMS configuration using Google Cloud
+headerTitle: Create a KMS configuration using Google Cloud
+linkTitle: Create a KMS configuration 
+description: Use YugabyteDB Anywhere to create a KMS configuration for Google Cloud KMS.
+aliases:
+  - /preview/yugabyte-platform/security/create-kms-config
+menu:
+  preview_yugabyte-platform:
+    parent: security
+    identifier: create-kms-config-1-google-kms
+    weight: 27
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="{{< relref "./aws-kms.md" >}}" class="nav-link">
+      <i class="fab fa-aws" aria-hidden="true"></i>
+      AWS KMS
+    </a>
+  </li>
+  <li >
+    <a href="{{< relref "./google-kms.md" >}}" class="nav-link active">
+      <i class="icon-google" aria-hidden="true"></i>
+      Google KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./azure-kms.md" >}}" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>
+      &nbsp;&nbsp;&nbsp;&nbsp;Azure KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./hashicorp-kms.md" >}}" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      HashiCorp Vault
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Google Cloud KMS. 
+
+Conceptually, Google Cloud KMS consists of a key ring containing one or more cryptographic keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key ring through [Google Cloud KMS](https://cloud.google.com/security-key-management).
```

Comment:
This is actually not necessary. If a key ring with the same name already exists, it validates and uses the existing key ring. Else, a new one is created automatically.

Reply:
Deleted the "Before..." sentence